sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -350,10 +351,23 @@ def test_enet_cv_positive_constraint():
     assert_true(min(enetcv_constrained.coef_) >= 0)
 
 
+def test_enet_cv_uniform_targets():
+    enet = ElasticNetCV()
```

Comment:
Can you please add tests for LassoCV. MultiTaskElasticNetCV and MultiTaskLassoCV also?